### DISCUSSION TITLE
Word exists in file in repository yet search does not find it

### DISCUSSION THREAD

**Discussion by skissane** [community/community#58306] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

This file: https://github.com/apache/openoffice/blob/trunk/main/ucb/source/ucp/odma/odma.h

Contains the word `ODM_API_VERSION`

And yet, when I search [`repo:apache/openoffice ODM_API_VERSION`](https://github.com/search?q=repo%3Aapache%2Fopenoffice%20ODM_API_VERSION&type=code) it only comes back with a single file, `main/ucb/source/ucp/odma/odma_provider.cxx`, which is not that one.

The `odma.h` file has been unmodified for 7 years, so this is not some kind of update recency issue.

**Comment by skissane** [comment]
Why is this labelled "more information needed"? What more information is needed?

**Reply by look** [reply]
This is part of GitHub's new system for categorizing product feedback so we can keep track of it. I've moved it to `in backlog`.

**Comment by look** [comment, marked as answer]
@skissane this file is in this ISO-8859 encoding (notably the `©` character, otherwise it would be equivalent to ASCII). It's [a limitation of our code search system](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations) that it only searches UTF-8 content, so this file is excluded. Sorry for the inconvenience, but the only way to fix this right now would be to remove that character. 

I've added your feedback to our tracking issue about this.
